npm-react-codemod/test/pure-render-mixin-test2.output.js

```diff
@@ -1,8 +1,9 @@
-var React = require('react/addons');
+var React = require('react');
+var shallowCompare = require('react/addons/shallowCompare');
 
 var MyComponent = React.createClass({
   shouldComponentUpdate: function(nextProps, nextState) {
-    return React.addons.shallowCompare(this, nextProps, nextState);
```

Comment:
Fixing up the tests; must have been the jscodeshift update by @fkling that made things a little more strict.

The code in these tests is not run. It verifies whether the transform works correctly. The purpose of this is to transform React.createClass calls to ES6 classes and this transform is one part of it. If we are suggesting people to use a different way to require react-addons we should update this codemod to also spit out code like this.